include/wlc/wlc.h

```diff
@@ -159,6 +159,13 @@ struct wlc_modifiers {
    uint32_t leds, mods;
 };
 
+/** Dispaly mode settings **/
+struct wlc_output_mode {
+   int32_t refresh;
+   int32_t width, height;
+   uint32_t flags; // WL_OUTPUT_MODE_CURRENT | WL_OUTPUT_MODE_PREFERRED
```

Comment:
Can you define a WLC enum for this? The values can match the wayland enum's values.